### PR TITLE
bincapz: rename to malcontent

### PR DESCRIPTION
**Pull request by egibs**
Related: https://github.com/chainguard-dev/malcontent/pull/464

#### For version bump PRs

- [x] The `epoch` field is reset to 0

### CODE REVIEW

**Comment by egibs** [comment]
@ajayk -- thanks!

File: malcontent.yaml [hunk 1]
```diff
@@ -38,11 +38,11 @@ test:
         - openssl
         - crane
   pipeline:
-    - name: Verify malcontent on itself
+    - name: Verify malcontent version
       runs: |
         set -o pipefail
-        output=$(mal analyze /usr/bin/mal)
```
Comment (egibs) [inline malcontent.yaml]: This output should be empty so I'll need to take a closer look at the regression. Either way, this takes too long and we removed this as a test in the `malcontent` repository.

Comment (egibs) [inline malcontent.yaml]: This output should be empty so I'll need to take a closer look at the regression. Either way, this takes too long and we removed this as a test in the `malcontent` repository.

Comment (egibs) [reply malcontent.yaml]: PR incoming to fix this behavior. I'll update this PR to the new hotfix release once it merges.

**Review by ajayk — APPROVED** [review]

**Comment by k4leung4** [comment]
should the bincapz package be withdrawn?
https://github.com/wolfi-dev/os/blob/main/WITHDRAWING_PACKAGES.md

**Pull request merged**